third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -608,6 +614,32 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
         });
     }
 
+    fn decl_ana_friend_decl(&mut self, friend_mod_id: &EA::ModuleIdent, friend: &EA::Friend) {
+        let address = match friend_mod_id.value.address {
```

Comment:
There is already a function `self.resolve_address` which you can use here. It will error on NamedUnassigned since it is expected that this has been resolved during expansion.